docs/core/porting/upgrade-assistant-wpf-framework.md

```diff
@@ -11,83 +11,434 @@ The [.NET Upgrade Assistant](upgrade-assistant-overview.md) is a command-line to
+[16:10:51 INF] Recommending TFM net6.0-windows because of dependency on project C:\code\migration\wpf\sampleApp\StarVoteControl\StarVoteControl.csproj
+C:\code\migration\wpf\sampleApp\WebSiteRatings\WebSiteRatings.csproj contains a reference to System.Web, which is not supported on .NET Core. You may have significant work ahead of you to fully port this project.
+C:\code\migration\wpf\sampleApp\WebSiteRatings\WebSiteRatings.csproj contains an App.config file. App.config is replaced by appsettings.json in .NET Core. You will need to delete App.config and migrate to appsettings.json if it's applicable to your project.
+[16:10:52 INF] Converting project C:\code\migration\wpf\sampleApp\WebSiteRatings\WebSiteRatings.csproj to SDK style
+[16:10:53 INF] Project file converted successfully! The project may require additional changes to build successfully against the new .NET target.
+[16:10:55 INF] Upgrade step Convert project file to SDK style applied successfully
+Please press enter to continue...
+```
+
+Pay attention to the output of each step. Notice that example output indicates that you'll have a manual step to complete after the upgrade:
+
+> App.config is replaced by appsettings.json in .NET Core. You will need to delete App.config and migrate to appsettings.json if it's applicable to your project.
+
+#### Move NuGet references
+
+Once the project format has been updated, the next step is to move the NuGet package references from the _packages.config_ file to the project file.
+
+```
+[16:55:18 INF] Applying upgrade step Clean up NuGet package references
+[16:55:18 INF] Removing outdated assembly reference: System.Configuration
+[16:55:18 INF] Removing outdated package reference: ControlzEx, Version=4.4.0
+[16:55:18 INF] Removing outdated package reference: Microsoft.Xaml.Behaviors.Wpf, Version=1.1.19
+[16:55:18 INF] Removing outdated package reference: SQLite.Native, Version=3.12.3
+[16:55:18 INF] Adding package reference: System.Configuration.ConfigurationManager, Version=5.0.0
+[16:55:18 INF] Adding package reference: Microsoft.DotNet.UpgradeAssistant.Extensions.Default.Analyzers, Version=0.3.261602
+[16:55:21 WRN] No version of System.Configuration.ConfigurationManager found that supports ["net452"]; leaving unchanged
+[16:55:21 INF] Upgrade step Clean up NuGet package references applied successfully
+Please press enter to continue...
+```
+
+#### Handle the TFM
+
+The tool next changes the [TFM](../../standard/frameworks.md) from .NET Framework to the suggested SDK. In this example, it's `net6.0-windows`.
+
+```
+[16:56:36 INF] Applying upgrade step Update TFM
+[16:56:36 INF] Recommending TFM net6.0-windows because of dependency on project C:\code\migration\wpf\sampleApp\StarVoteControl\StarVoteControl.csproj
+[16:56:40 INF] Updated TFM to net6.0-windows
+[16:56:40 INF] Upgrade step Update TFM applied successfully
+Please press enter to continue...
+```
+
+#### Upgrade NuGet packages
 
 Next, the tool updates the project's NuGet packages.
```

Comment:
Since this looks similar to the previous step that changed NuGet references, it might help to explain that this step is about updating versions to work with the new TFM (now that the project is targeting .NET 6).